README.md

```diff
@@ -38,11 +42,30 @@ Common packages among various go-btfs projects.
 │   └── status
 │       ├── status.pb.go
 │       └── status.proto
-└── prototool.yaml
+├── prototool.yaml
+├── tree.txt
+└── utils
+    ├── grpc_health_check_provider.go
+    ├── runtime.go
+    └── runtime_test.go
 ```
 
 ## Install Tools
 
+#### install TRON version(pg support) protobuf
+
+```
+cd github.com/tron-us
+```
+```
+git clone https://github.com/TRON-US/protobuf
+```
+```
+cd protobuf && build
```

Comment:
How is the protogen replaced? This line only builds..

Also, this should be part of the `make install` to make it convenient for other devs

Reply:
good catch, it should be make instead which will build and install(replace protoc-gen-gogo)